auth-web/src/App.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <v-app id="app">
     <div class="header-group" ref="headerGroup">
-      <sbc-header :key="$route.fullPath"></sbc-header>
+      <sbc-header
+        :key="$route.fullPath"
+        :account-name="currentOrganization && currentOrganization.name"
+        :pending-approval-count="pendingActionCount"></sbc-header>
```

Comment:
Just some thoughts here : since we are passing the values to header, may not work in cases where the user is in another application (coops, create-business, ppr etc.) and won't see the changes when someone else is working on the notification actions from another browser. Also with the new strategy where the applications can work standalone, this solution would cause some issues I guess. May be we should think about making the header stands by own, rather than passing the values from parent application.

Reply:
Sorry @pksumesh I didn't see these comments until now.  I'm going to create a ticket to address this by adding a service layer in the common component so that it can make calls to our API as required.